djangocms_oidc/constants.py

```diff
@@ -1,2 +1,2 @@
 DJNAGOCMS_PLUGIN_SESSION_KEY = 'djangocms_oidc_plugin_id'
-DJNAGOCMS_USER_SESSION_KEY = 'dinagocms_oidc_user_info'
+DJNAGOCMS_USER_SESSION_KEY = 'djnagocms_oidc_user_info'
```

Comment:
There is also a typo in `DJNAGOCMS_`.

Reply:
Typo fixed and an order in credentials too.